tests/lib/ExpectObj.php

```diff
@@ -15,7 +15,7 @@ public function toMatchExpectFile(string $file): void where T = string {
 
   public function toMatchExpectregexFile(string $file): void where T = string {
     $this->toMatchRegExp(
-      \file_get_contents($file),
+      '/^'.\file_get_contents($file).'$/s',
```

Comment:
How did this pass before ? :/ Did we have no expectregexp files?

Reply:
Previously this was only called from `testExamplesTypecheck`, and we have no typechecker expectregex files.